src/driver/experiment_driver.py

```diff
@@ -102,20 +106,22 @@ def __init__(
         output_file: str,
         intermediate_dir: str,
         random_seed: int,
+        cores: int,
     ):
         self._data_design_dir = data_design_dir
         self._experimental_design = experimental_design
         self._output_file = output_file
         self._intermediate_dir = intermediate_dir
         self._rng = np.random.default_rng(seed=random_seed)
         np.random.seed(random_seed)
+        self._cores = cores
```

Comment:
Please modify the unit tests to accommodate the changes. Also, update the README if necessary.

Reply:
I have added a unit test.  I will update the README in a separate PR.